src/LaboratoryResults-3-0/Test/_reference/sendreceive_request/lr-loa-TEST-LOINC-Scenario1-1.xml

```diff
@@ -1,24 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Bundle xmlns="http://hl7.org/fhir">
-   <id value="lr-slr-TEST-Scenario4a-4a"/>
+   <id value="lr-loa-TEST-LOINC-Scenario1-1"/>
    <type value="transaction"/>
    <entry>
-      <fullUrl value="urn:uuid:6ade89b6-5c3f-11ed-4420-020000000000"/>
+      <fullUrl value="urn:uuid:66a565dc-5d41-11ee-7831-020000000000"/>
       <resource>
          <Observation>
             <meta>
                <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-LaboratoryTestResult"/>
             </meta>
-            <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-CopyIndicator">
-               <valueBoolean value="true"/>
-            </extension>
-            <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-EdifactReferenceNumber">
-               <valueString value="111666"/>
-            </extension>
-            <identifier>
```

Comment:
Verwachten we niet iets waar het volgende gemeld wordt:
Laboratorium uitslag identificatie | 11081991 (in identificerend systeem: 2.16.840.1.113883.2.4.3.11.999.25.4296.4)